### ISSUE TITLE
"Automatic Mode" interferes with Firefox's builtin Address Bar tab search feature ("%" search term)

### ISSUE THREAD

**Issue by spiralsam** [stoically/temporary-containers#461]
Normally Firefox allows the user to search any open tabs across all windows by typing "% <title to search for>" in the Address Bar, then switch to it by selecting the result in the list.

Turning on "Automatic Mode" interferes with this, even though it shouldn't. It looks as though using "%" will only search tabs also from in the currently-active-tab's container group, making the feature significantly less useful.

**Comment by stoically** [comment]
Yeah, that's unfortunate, please see: https://bugzilla.mozilla.org/show_bug.cgi?id=1500991
